presidio-analyzer/presidio_analyzer/validation/validation_utils.py

```diff
@@ -0,0 +1,23 @@
+from typing import List, Tuple
+
+
+class ValidationUtils:
+    """
+    Utility functions for Presidio Analyzer.
+
+    The class provides a bundle of utility functions that help centralizing the
+    logic for re-usability and maintainability
+    """
+
+    @staticmethod
+    def sanitize_value(text: str, replacement_pairs: List[Tuple[str, str]]) -> str:
```

Comment:
Should this be in "utils" or can this be a protected function in `PatternRecognizer` or `EntityRecognizer`? Do we foresee uses to this outside of the pattern detection flow?